config/routes.rb

```diff
@@ -91,4 +91,5 @@
 
   resources :contacts, only: %i[new create]
   resources :feedback, only: %i[new create]
+  get "feedback", to: "feedback#show"
```

Comment:
Adding show to resources would require an id. This creates a route to the show action without an id as we only have one thing to show.

Reply:
Ah, we could set it `resource` instead of `resources` ?

Reply:
Ah, right! Update - Using bootstrap, we no longer need a show action/route.